File: model/Laboratorio.php
<?php

class Laboratorio {

    public function __construct(
        public ?int $id = null,
        public string $nome,
        public string $sigla,
        public string $bloco,
        public int $sala,
    ) {
        $this->id       = $id;
        $this->nome     = $nome;
        $this->sigla    = $sigla;
        $this->bloco     = $bloco;
        $this->sala     = $sala;
    }
}

File: python-api/model/Laboratorio.py
from dataclasses import dataclass
from typing import Optional

from model.Bloco import Bloco

@dataclass
class Laboratorio:
    id: Optional[int]
    nome: str
    sigla: str
    id_bloco: int|Bloco
    sala: int
    ativo: Optional[bool] = True
